### PR TITLE
binance: new API documentation

### PR DESCRIPTION
**Pull request by Dan-krm**
Updated the binance documentation links

### CODE REVIEW

File: doc/ccxt.pro.manual.rst [hunk 1]
```diff
@@ -117,7 +117,7 @@ The CCXT Pro library currently supports the following 46 cryptocurrency exchange
      - binance
      - `Binance <https://accounts.binance.com/en/register?ref=D7YA7CLY>`__
      - .. image:: https://img.shields.io/badge/*-lightgray
-          :target: https://binance-docs.github.io/apidocs/spot/en
```
Comment (carlosmiei) [inline doc/ccxt.pro.manual.rst:120]: .rst files are automatically enerated

File: ts/src/pro/binanceusdm.ts [hunk 2]
```diff
@@ -19,6 +19,7 @@ export default class binanceusdm extends binance {
                 'defaultSubType': 'linear',
             },
             // https://binance-docs.github.io/apidocs/futures/en/#error-codes
+            // https://developers.binance.com/docs/derivatives/usds-margined-futures/error-code
```
Comment (carlosmiei) [inline ts/src/pro/binanceusdm.ts:22]: @Dan-krm can we add the docs to the pro/binance.ts?

Comment (Dan-krm) [reply ts/src/pro/binanceusdm.ts:23]: Yeah I added a doc link to the pro implementations

File: ts/src/binance.ts [hunk 3]
```diff
@@ -2598,9 +2598,9 @@ export default class binance extends Exchange {
          * @method
          * @name binance#fetchTime
          * @description fetches the current integer timestamp in milliseconds from the exchange server
-         * @see https://binance-docs.github.io/apidocs/spot/en/#check-server-time       // spot
-         * @see https://binance-docs.github.io/apidocs/futures/en/#check-server-time    // swap
-         * @see https://binance-docs.github.io/apidocs/delivery/en/#check-server-time   // future
+         * @see https://developers.binance.com/docs/binance-spot-api-docs/rest-api#check-server-time                            // spot
```
Comment (carlosmiei) [inline ts/src/binance.ts:2601]: @Dan-krm we still have hundreds of links using `binance-docs`, don't we need to update them as well?

image

Comment (Dan-krm) [reply ts/src/binance.ts:2601]: I was planning on updating the rest of the links in the method docstrings in a separate PR but I can just add those changes to this PR

Comment (Dan-krm) [reply ts/src/binance.ts:2600]: I've finished updating the docstring links

Comment (carlosmiei) [reply ts/src/binance.ts:2600]: got it, thanks

**Pull request merged**